dmsite/venv/src/Profile.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Created by christinadaley on 1/17/19.
+ */
+
+import React from 'react';
+import { Container, ListGroup, ListGroupItem, Card, CardBody, CardTitle, Button } from 'reactstrap';
+import '/.style.css';
+
+class Profile extends React.Component {
+
+    render() {
+        return (
+            <div id="div">
+                <Card id="card">
+                    <h1 align="center"> Campaigns </h1>
+                    <CardTitle className="pl-4 pt-4">{this.props.name}</CardTitle>
+                    <CardBody>
+                        <ListGroup align="center" className="campaign-list" flush>
+                            <ListGroupItem>Campaign 1</ListGroupItem>
+                            <ListGroupItem>Campaign 2</ListGroupItem>
+                            <ListGroupItem>Campaign 3</ListGroupItem>
+                        </ListGroup>
```

Comment:
When we want to actually display the list of campaigns, let me know what you need so I can set that up and you can just make a request and parse the data returned.

Reply:
Okay I will check that Dustyn isn't working on this because this was originally his page!